discern.lic

```diff
@@ -0,0 +1,42 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#Discern
+=end
+
+custom_require.call(%w[common common-travel events spellmonitor drinfomon])
+
+class AutoDiscern
+  include DRC
+  include DRCA
+
+  def initialize
+    settings = get_settings
+    UserVars.discerns = {} if settings.force_discerns

```

Comment:
I'm not sure what I think of `force_discerns` being a setting. That implies that you want to re-discern every time this script is run. Is that a common use case?

This feels more like a script argument to me, so you can specify it (or not) differently each time the script is run. What do you think?

Reply:
grumble no I thought I removed the force setting it was only for me to test with.  i figure you can run ;vars delete discerns instead of this setting